popmon/visualization/templates/core.html

```diff
@@ -16,10 +16,11 @@
     {% include 'assets/css/bootstrap.min.css' %}
     {% include 'assets/css/custom-style.css' %}
   </style>
+  
 
 </head>
 <body id="page-top">
-
+  <script src="https://cdn.plot.ly/plotly-2.12.1.min.js"></script>
```

Comment:
Should we add this js file to our assets/ instead? 
(So there are no external dependencies.)

Reply:
I would say either keep it this way or embed all the `JavaScript` code into the HTML file because the report would not work as a standalone HTML file if we put JS code in assets/. The drawback of adding the `CDN` source (the way Pradyot did) is that the report would not work without the internet, on the contrary, embedding this `JS` code into the `HTML` code itself would make the `HTML` code unreadable.

Reply:
Actually that's what I meant: embed all the JavaScript code into the HTML file. So the report also works in an environment with limited internet access, such as DAP. We could also make a switch for this (so, make it optional) when the report is written out to file.

Reply:
Agreed, I had something similar in mind regarding the optional parameter.

Reply:
Yes I think that's a nice solution here.

Reply:
Initially I had the js embedded in the HTML which would allow report to be visible offline but embedding it means 4MB extra file size. But it does make sense to have it available offline and doing it with parameter would be neat.